benchmark/DriverBench/SingleDocBench.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace MongoDB\Benchmark\DriverBench;
+
+use Generator;
+use MongoDB\Benchmark\BaseBench;
+use MongoDB\BSON\Document;
+use MongoDB\Driver\Command;
+use MongoDB\Driver\WriteConcern;
+use PhpBench\Attributes\BeforeMethods;
+use PhpBench\Attributes\ParamProviders;
+use PhpBench\Attributes\Revs;
+
+use function array_map;
+use function file_get_contents;
+use function range;
+
+/** @see https://github.com/mongodb/specifications/blob/ddfc8b583d49aaf8c4c19fa01255afb66b36b92e/source/benchmarking/benchmarking.rst#single-doc-benchmarks */
+#[BeforeMethods('prepareDatabase')]
+final class SingleDocBench extends BaseBench
+{
+    public function prepareDatabase(): void
+    {
+        self::getCollection()->drop();
+    }
+
+    /** @see https://github.com/mongodb/specifications/blob/ddfc8b583d49aaf8c4c19fa01255afb66b36b92e/source/benchmarking/benchmarking.rst#run-command */
+    public function benchRunCommand(): void
+    {
+        $manager = self::createClient()->getManager();
+
+        for ($i = 0; $i < 10_000; $i++) {
+            $manager->executeCommand(self::getDatabase(), new Command(['hello' => true]));
```

Comment:
You could extract the method call for benchmarking purposes. 10k method calls will add a certain overhead which we don't want to include in our testing. Command creation can be in the loop as that would usually be part of this kind of workflow.

```suggestion
        $manager = self::createClient()->getManager();
        $database = self::getDatabase();

        for ($i = 0; $i < 10_000; $i++) {
            $manager->executeCommand($database, new Command(['hello' => true]));
```